cli.py

```diff
@@ -0,0 +1,49 @@
+import logging
+import os
+import sys
+
+import click
+import daiquiri
+from pythonjsonlogger.jsonlogger import JsonFormatter
+
+from exporters import DataDogExporter
+from receiver import CeleryEventReceiver
+from store import InMemoryStore
+
+
+def setup_logging():
+    prod_log_format = (
+        "%(asctime)s [%(process)d] %(levelname)-8.8s %(name)s: %(message)s"
+    )
+    is_production_env = os.environ.get("PROD", "False").lower() == "true"
+    output_formatter = (
+        JsonFormatter(prod_log_format)
+        if is_production_env
+        else daiquiri.formatter.ColorFormatter()
```

Comment:
In the future we should extract this to a "centralied" config.py(or settings.py). For now it doesn't matter since there's only 2 things to configure. But if this different files starts to get littered with os.environ or is_prod etc. then it's time to centralize.

Reply:
yep, will keep this in mind for future PRs